control/grpc.py

```diff
@@ -620,7 +643,7 @@ def delete_subsystem_safe(self, request, context):
                 )
                 self.subsys_ha.pop(request.subsystem_nqn)
                 self.subsys_max_ns.pop(request.subsystem_nqn)
-                self.logger.info(f"delete_subsystem {request.subsystem_nqn}: {ret}")
+                self.logger.debug(f"delete_subsystem {request.subsystem_nqn}: {ret}")
```

Comment:
Should this be any exception level, since it hit an exception?

Reply:
@pcuzner this is not an exception, this is the return value from spdk. The except clause is two lines down and there we use self.logger.exception().